qhdopt/qhd_base.py

```diff
@@ -90,6 +129,7 @@ def ionq_setup(
         time_discretization: int = 10,
         gamma: float = 5,
         on_simulator: bool = False,
+        backend: str = "Harmony", # EDIT: Added backend parameter
```

Comment:
This should be included in a separate PR as it is unrelated to this change. Create a separate PR